Exdi/exdigdbsrv/GdbSrvControllerLib/GdbSrvControllerLib.cpp

```diff
@@ -187,8 +200,18 @@ class GdbSrvController::GdbSrvControllerImpl
             dataBuffer.insert(dataBuffer.end(), 1, NumberToAciiHex((pMonitorCmd[idx] & 0xf)));
         }
 
-        std::string commandMonitor("qRcmd,");
-        commandMonitor += dataBuffer.c_str();
+        std::string commandMonitor;
+        if (strstr(pMonitorCmd, g_RequestGdbSetReadPAmode) != nullptr ||
+            strstr(pMonitorCmd, g_RequestGdbSetWritePAmode) != nullptr)
+        {
+            commandMonitor += pMonitorCmd;
+        }
+        else
+        {
+            commandMonitor += "qRcmd,";
+            commandMonitor += dataBuffer.c_str();
+            commandMonitor += dataBuffer.c_str();
+        }

```

Comment:
Appending twice? If this is intentional, it deserves a comment.

Reply:
Good catch. The first issue will regress telemetry on exdiGDBSrv